src/App.js

```diff
@@ -0,0 +1,108 @@
+// import "./App.scss";
+import "./styles/main.scss";
+import Header from "./components/Header";
+import AddNewInput from "./components/AddNewInput";
+import Filter from "./components/FilterSortItems";
+import CheckListItem from "./components/CheckListItem";
+import React, { Component } from "react";
+
+export default class App extends Component {
+  constructor(props) {
```

Comment:
![image](https://user-images.githubusercontent.com/22271897/119938617-6a156800-bfac-11eb-8f7d-a86ca74d0430.png)
You can only do this as well.